source/material_model/rheology/visco_plastic.cc

```diff
@@ -512,6 +522,10 @@ namespace aspect
                            "dynamic stresses are much higher than the lithostatic pressure. "
                            "If false, the minimum pressure in the plasticity formulation will "
                            "be set to zero.");
+        prm.declare_entry ("Use adiabatic pressure in creep viscosity", "false",
+                           Patterns::Bool (),
+                           "Whether to use adiabatic pressure instead of full pressure (default) "
```

Comment:
Can you expand the documentation here to explain why this option would be useful?